airtableClient.js

```diff
@@ -34,6 +40,16 @@ module.exports = {
 				throw new Error("Error fetching Airtable records" + error)
 		  });
 
+		return result
+	},
+
+	getHours: async function() {
```

Comment:
You may want to switch this to the "Declare then assign" we talked about in DMs at the end of January.  It shouldn't require using `rewire` to test, everything else should still be the same.

I think :)

Reply:
I thought if we knew it had to be exported anyway, it was fine to do like this? I'm not opposed, just trying to understand :) I looked back at our convo - is this because of the hoisting stuff? Should `getMutualAidSites` also be switched then?

Reply:
Responding a bit out of order because the most important bit (perspective as opposed to reason) is actually last.

>  I looked back at our convo - is this because of the hoisting stuff?

Yeah.

After thinking about it (afterwards, mind) I realized that the hoisting is a good thing.  One of the weirder things that can pop up is the declaration order as a debugging issue.  Rather than worry about that as well, on top of "Javascript can be a bit weird, even at the most vanilla of times", it's just one less thing to deal with and the only cost is dropping an `=`.

> Should getMutualAidSites also be switched then?

It would look weird otherwise =) 

> I thought if we knew it had to be exported anyway, it was fine to do like this? I'm not opposed, just trying to understand :)

To be honest, this is one of those low-key neurosis things that causes people to get super invested in linters and whatnot (i.e. the "pick one way and stick to it consistently" thing).  But I do think it has a value (as mentioned above).

Reply:
for sure, I don't really have a preference. Does it work for me to switch all the functions I add in this PR to be that form of declaration and then do a folllowup to clean up other functions that are already here, or would you prefer I lump those changes into this PR?

Reply:
Totally fine to clean it up later.